common/src/main/resources/application.conf

```diff
@@ -9,3 +9,7 @@ spray.can {
     verbose-error-messages = "on"
   }
 }
+
+spray.can.server {
+  request-timeout = 35s
+}
```

Comment:
I updated my branch with this change, but for some reason this PR is not updating to reflect the [new commit](https://github.com/mars/incubator-predictionio/commits/extend-request-timeout). Assuming it's a Github glitch which will eventually fix itself.